index.html

```diff
@@ -0,0 +1,241 @@
+                            <li><a href="#">English</a></li>
+                            <li><a href="#">My Page</a></li>
+                            <li><a href="#">Logout</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+            <div class="navigation_wrapper hide--mobile">
+                <nav class="navbar--mobile navbar--desktop">
+                    <div class="logo_container hide--mobile">
+                        <a href="./index.html">
+                            <img class="logo" src="./assets/images/logo.png" alt="logo company">
+                        </a>
+                    </div>
+                    <ul class="menu_list">
+                        <li>
+                            <a href="./about.html">About</a>
+                        </li>
+                        <li>
+                            <a href="#">Program</a>
+                        </li>
+                        <li>
+                            <a href="#">Join</a>
+                        </li>
+                        <li>
+                            <a href="#">Sponsor</a>
+                        </li>
+                        <li>
+                            <a href="#">News</a>
+                        </li>
+                        <li class="cc_campaign">
+                            <a href="#">
+                                CC Campaign
+                            </a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </section>
+    </header>
+
+    <section class="headline_wrapper">
+        <div class="headline">
+            <div class="title_wrapper">
+                <div class="suport_title">
+                    <p>"Hello! Coders"</p>
+                </div>
+                <div class="title">
+                    <h1>
+                        CREATIVE CODERS GLOBAL SUMMIT 2023
+                    </h1>
+                </div>
+            </div>
+        </div>
+
+        <div class="description_wrapper">
+            <div class="description">
+                <p>A joyful celebration believing in the value of openness and sharing, creating a positive change with
+                    people from over 80 countries is taking place in October, in Korea.</p>
+            </div>
+        </div>
+
+        <div class="day_address_wrapper">
+            <div class="day_address">
+                <p class="day">2023.10.18(THU) ~ 25(FRI)</p>
+                <p class="address"> @ National Museum of Charlemagne, Application demo and more </p>
+            </div>
+        </div>
+    </section>
+
+    <section class="main_program_wrapper">
+        <div class="main_program">
+            <div class="main_program_title_wrapper">
+                <h2 class="main_program_title">Main Program</h2>
+                <hr class="main_program_bar">
+            </div>
```

Comment:
- [OPTIONAL] Your `<div class="day-address">` is overlapped by the main-program background. Kindly look into it and fix it.

### Desktop
![your-main-program-desktop](https://user-images.githubusercontent.com/26597432/210702342-03867d4d-16fe-464c-a221-bcb5202b2a1f.png)

### Mobile 
![your-home-mobile](https://user-images.githubusercontent.com/26597432/210702360-61fe5b17-c53c-4b8f-aab8-9c2a948febcb.png)


#### Guiding Template Desktop
![temp-home-desktop](https://user-images.githubusercontent.com/26597432/210702645-6b59fc9c-b76d-4570-93de-0b46baff65fa.png)

#### Guiding Template Mobile
![temp-home-mobile](https://user-images.githubusercontent.com/26597432/210702670-fac24b02-6093-49c4-865a-f3f6778476a8.png)